core/src/test/scala/org/ensime/indexer/ClassfileIndexerSpec.scala

```diff
@@ -3,42 +3,35 @@
 package org.ensime.indexer
 
 import akka.event.slf4j.SLF4JLogging
+import org.ensime.fixture.IsolatedEnsimeVFSFixture
+import org.ensime.util.EnsimeSpec
 import org.scalatest.{ BeforeAndAfterAll, FunSpec, Matchers }
+import org.ensime.vfs._
 
-class ClassfileIndexerSpec extends FunSpec with Matchers with ClassfileIndexer with SLF4JLogging
-    with BeforeAndAfterAll {
+class ClassfileIndexerSpec extends EnsimeSpec with IsolatedEnsimeVFSFixture {
 
-  var vfs: EnsimeVFS = _
+  val indexer = new ClassfileIndexer with SLF4JLogging {}
+  import indexer._
 
-  override def beforeAll(): Unit = {
-    vfs = EnsimeVFS()
+  // TODO: some assertions (currently we're just checking that no exceptions are raised!)
```

Comment:
(I recommend reading each commit individually this time because there is one that just moves files around)